.taskcluster.yml

```diff
@@ -59,7 +58,7 @@ tasks:
 ###############################################################################
   - $if: 'tasks_for == "github-push"'
     then:
-      $if: 'event.ref == "refs/heads/master"'
+      $if: 'event.repository.fork == false'
```

Comment:
This event is documented at https://developer.github.com/v3/activity/events/types/#pushevent